src/views/DataExploration/components/PageContent/tabs/Summary/AgeAtDiagnosisGraphCard/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { RawAggregation } from 'graphql/models';
+import { toChartData } from 'utils/charts';
+import BarChart from 'components/uiKit/charts/Bar';
+import GridCard from '@ferlab/ui/core/view/v2/GridCard';
+import { truncateString } from 'utils/string';
+import { INDEXES } from 'graphql/constants';
+import { useHistory } from 'react-router-dom';
+import { ArrangerValues } from '@ferlab/ui/core/data/arranger/formatting';
+import { isEmpty } from 'lodash';
+import Empty from '@ferlab/ui/core/components/Empty';
+import { AGE_AT_DIAGNOSIS_QUERY } from 'graphql/summary/queries';
+import { ARRANGER_API_PROJECT_URL } from 'provider/ApolloProvider';
+import useApi from 'hooks/useApi';
+import useParticipantResolvedSqon from 'graphql/participants/useParticipantResolvedSqon';
+import CardHeader from 'views/Dashboard/components/CardHeader';
+import intl from 'react-intl-universal';
+import { DATA_EXPLORATION_QB_ID } from 'views/DataExploration/utils/constant';
+import { updateActiveQueryField } from '@ferlab/ui/core/components/QueryBuilder/utils/useQueryBuilderState';
+import { RangeOperators } from '@ferlab/ui/core/data/sqon/operators';
+
+interface OwnProps {
+  id: string;
+  className?: string;
+}
+
+type DiagnosisQueryResult = {
+  data: {
+    [key: string]: {
+      [key: string]: {
+        total: number;
+      };
+    };
+  };
+};
+
+const transformAgeAtDiagnosis = (results: DiagnosisQueryResult) =>
+  Object.keys(results?.data?.participant || {}).map((key) => ({
+    id: intl.get(`screen.dataExploration.tabs.summary.ageAtDiagnosis.${key}`),
+    label: key,
+    value: results?.data?.participant[key]?.total || 0,
+  }));
+
+const graphSetting: any = {
```

Comment:
find a better type than `any` of leave no type (Typescript should infer it)

Reply:
In this case, it seems needed

```
Unlike unknown, variables of type any allow you to access arbitrary properties, even ones that don’t exist. These properties include functions and TypeScript will not check their existence or type:
```
If I don't infer it, I ended up with the following error.
![image](https://user-images.githubusercontent.com/65532894/197589217-ec9f98e5-c73e-4e01-b440-7395509d7d51.png)